lib/galaxy/tool_util/deps/container_classes.py

```diff
@@ -235,7 +235,7 @@ def add_var(name, value):
                 # and Galaxy.
                 defaults += ",$tmp_directory:/tmp:rw"
             else:
-                defaults += ",$_GALAXY_JOB_TMP_DIR:rw"
+                defaults += ",$_GALAXY_JOB_TMP_DIR:rw,$TMPDIR:rw,$TMP:rw,$TEMP:rw"
```

Comment:
Do we need to check if these variables exist / are defined?

Reply:
We can't, but if they don't exist nothing happens, try `docker run -it -v "::rw" busybox sh`

Reply:
Cool 😎